### PR TITLE
Sensor messages

### PR DESCRIPTION
**Pull request by rafalgajda-silvair**
This patch introduces constructs for sensor messages. 
There is still some work to be done for the future to make sure all fields are parsed according to specification.

### CODE REVIEW

File: bluetooth_mesh/messages/sensor.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+FastCadencePeriodDivisorAndTriggerType = EmbeddedBitStruct(
+    "status_trigger_type" / BitsInteger(1),
+    "fast_cadence_period_divisor" / BitsInteger(7)
+)
+
+UnitlessTriggerDelta = DefaultCountValidator(Int8ul, rounding=1, resolution=0.1)
+
+TriggerDelta = Struct(
+    Switch(
+        this.status_trigger_type,
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/messages/sensor.py]: Make this an enum (using `EnumAdapter`)?

File: bluetooth_mesh/messages/properties.py [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+)
+
+RelativeRuntimeInAGenericLevelRange = Struct(
+    "relative_value" / DefaultCountValidator(Int8ul, rounding=1, resolution=0.5),
+    "minimum_generic_level" / DefaultCountValidator(Int16ul),
+    "maximum_generic_level" / DefaultCountValidator(Int16ul),
+)
+
+
+def FixedString(size):
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/messages/properties.py]: `PaddedString`?

File: bluetooth_mesh/messages/sensor.py (hunk 1 above)
Comment (rafalgajda-silvair) [reply bluetooth_mesh/messages/sensor.py]: Not sur if that would be easier. 
status_trigger_type is taken from FastCadencePeriodDivisorAndTriggerType field which is an EmbeddedBitStruct on the previous octet.

File: bluetooth_mesh/messages/properties.py (hunk 2 above)
Comment (rafalgajda-silvair) [reply bluetooth_mesh/messages/properties.py]: Done

**Pull request merged**